tests/browser/test_webelem.py

```diff
@@ -87,17 +86,17 @@ def _style_property(name, strategy):
     return wrapped
 
 
-class WebElementWrapperTests(unittest.TestCase):
+class TestWebElementWrapper:
```

Comment:
I'm skipping the **new-style-class** comments for the rest of the diff :exclamation: